### DISCUSSION TITLE
Can't login on multiple devices with one user account at the same time?

### DISCUSSION THREAD

**Discussion by KLA6** [laravel/framework#35006] [Q&A]
### Environment
* Laravel 8.4.0 with Inertia
* PHP 7.4.11 

### Question
My core user login part is just simple as the below.
```
$user_db = User::where( 'email', $user_oauth->getEmail() )->first(); // $user_oauth is from Socialite.
Auth::login( $user_db, true );
```
BTW, it seems from Laravel 8, the default login behavior is PREVENTING logins on multiple devices with one user account at the same time. It means that...
* I logged in on my PC.
* And then, I logged in on my mobile phone.
* And then, if I refresh on my PC, it requests to login again.

Is there any way to ALLOW logins on multiple devices with one user account at the same time?

**Comment by ndroo** [comment]
Did you ever resolve this? Same issue here
